example-apps/internal-knowledge-search/app-ui/.env

```diff
@@ -0,0 +1,3 @@
+SEARCH_APP_NAME=some-search-application
```

Comment:
Better to rename in .env.example as common practise for this kind of file

Reply:
Will add this as an improvement issue for later, thanks!